Formula/ghostscript.rb

```diff
@@ -1,15 +1,16 @@
 class Ghostscript < Formula
   desc "Interpreter for PostScript and PDF"
   homepage "https://www.ghostscript.com/"
-  url "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs9550/ghostpdl-9.55.0.tar.gz"
-  sha256 "b73cdfcb7b1c2a305748d23b00a765bcba48310564940b1eff1457f19f898172"
+  url "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs9561/ghostpdl-9.56.1.tar.xz"
+  sha256 "05e64c19853e475290fd608a415289dc21892c4d08ee9086138284b6addcb299"
   license "AGPL-3.0-or-later"
 
   # We check the tags from the `head` repository because the GitHub tags are
   # formatted ambiguously, like `gs9533` (corresponding to version 9.53.3).
   livecheck do
-    url :head
-    regex(/^ghostpdl[._-]v?(\d+(?:\.\d+)+)$/i)
+    url "https://github.com/ArtifexSoftware/ghostpdl-downloads"
+    strategy :page_match
+    regex(/^ghostpdl(?:\s)?v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    url :stable
    regex(/href=.*?ghostpdl[._-]v?(\d+(?:\.\d+)+)\.t/i)
    strategy :github_latest
```

The `README` is very limited in the information it provides, so it would be better to check the GitHub releases. Let's start by checking the "latest" release and matching the version from the tarball we use in the `stable` URL. If we run into issues with this in the future (e.g., a release doesn't include a GhostPDL tarball), we could make an exception and check the GitHub releases page directly (though we try to avoid this whenever possible, since the pagination can cause problems):

```ruby
    url "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases?q=prerelease%3Afalse"
    regex(/href=.*?ghostpdl[._-]v?(\d+(?:\.\d+)+)\.t/i)
    strategy :page_match
```